src/applications/personalization/search-representative/sass/search-representative.scss

```diff
@@ -1,3 +1,16 @@
+[data-chapter="basicInformation"] .review-row > dt {
```

Comment:
I think something like this might be better, because I'm seeing all 'Edit' buttons hidden and the row text in other panels is misaligned currently
```css
[data-chapter="basicInformation"] {
  dt {
    display: none;
  }

  .review-row {
    flex-direction: column !important;
    padding: 2.5rem;
  }

  .edit-btn {
    display: none;
  }
}
```